endpoints/jwt_test.go

```diff
@@ -113,7 +113,7 @@ const googCerts = `{
 	}]
 }`
 
-func TestverifySignedJWT(t *testing.T) {
+func TestVerifySignedJWT(t *testing.T) {
```

Comment:
That's what [testing](http://golang.org/pkg/testing/) page says. A test method should start with `TestXxx` where `Xxx` can be any alphanumeric string but the first letter must not be in [a-z].

I think this test wasn't even running. But since the change touches on authentication, I thought it would be a good time to fix it.